deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml

```diff
@@ -80,33 +80,145 @@ spec:
+              - 'urn:alm:descriptor:io.kubernetes:Secret'
+              - 'urn:alm:descriptor:com.tectonic.ui:fieldDependency:security.endpointAuthentication:true'
+          - displayName: type
+            description: Service type
+            path: service.type
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:select:Cache'
+              - 'urn:alm:descriptor:com.tectonic.ui:select:DataGrid'
+          - displayName: Remote site secret name
+            description: Specifies the access secret for remote site
+            path: service.sites.locations[].secretName
+            x-descriptors:
+              - 'urn:alm:descriptor:io.kubernetes:Secret'
+          - displayName: Node port
+            description: Node port to which the load balancer service forwards traffic
+            path: service.sites.local.expose.nodePort
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:number'
+              - 'urn:alm:descriptor:com.tectonic.ui:fieldDependency:service.sites.local.expose.type:NodePort'
+          - displayName: Cache replication factor
+            description: Configures replicas for each cache entry
+            path: service.replicationFactor
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:number'
+          - description: Enable/disable container ephemeral storage
+            displayName: Container ephemeral storage
+            path: service.container.ephemeralStorage
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
+          - displayName: Storage size
+            description: The size of the storage to use in the persistent volume claim
+            path: service.container.storage
+            x-descriptors:
+              - 'urn:alm:descriptor:text'
+          - displayName: Storage class name
+            description: The name of the storage class object to use for the persistent volume claim
+            path: service.container.storageClassName
+            x-descriptors:
+              - 'urn:alm:descriptor:io.kubernetes:StorageClass'
+              - 'urn:alm:descriptor:com.tectonic.ui:fieldDependency:service.container.ephemeralStorage:false'
+          - displayName: Expose host name for the Route
+            description: Specifies the hostname where Infinispan is exposed
+            path: expose.host
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:text'
+              - 'urn:alm:descriptor:com.tectonic.ui:fieldDependency:expose.type:Route'
+          - displayName: Persistent volume claim name
+            description: Name of the persistent volume claim with custom libraries
+            path: dependencies.volumeClaimName
+            x-descriptors:
+              - 'urn:alm:descriptor:io.kubernetes:PersistentVolumeClaim'
         statusDescriptors:
           - description: The current pods
             displayName: Pods Status
```

Comment:
```suggestion
            displayName: Pod Status
```

Reply:
This is not single Pod status, it's pie-chart diagram for all of the single Infinispan CR Pods